### PR TITLE
[FIX] mail: Fix failed tests (message and chat)

### PR DESCRIPTION
**Pull request by maki-odoo**

### CODE REVIEW

File: addons/mail/static/tests/qunit_suite_tests/widgets/form_renderer_tests.js [hunk 1]
```diff
@@ -59,10 +59,10 @@ QUnit.module("mail", {}, function () {
                 "button:contains(Activities)",
                 "there should be a schedule activity button"
             );
-            assert.containsNone(
+            assert.containsOnce(
```
Comment (seb-odoo) [inline addons/mail/static/tests/qunit_suite_tests/widgets/form_renderer_tests.js]: skipped test should not be adapted this way

File: addons/mail/static/tests/new/chat/chat_window_tests.js [hunk 2]
```diff
@@ -700,11 +700,9 @@ QUnit.test(
         }
         await start();
         await insertText(".o-mail-composer-textarea", "WOLOLO");
-        await afterNextRender(() =>
-            triggerEvent(target.querySelector(".o-mail-composer-textarea"), "keydown", {
-                key: "Enter",
-            })
-        );
+        await triggerEvent(target, ".o-mail-composer-textarea", "keydown", {
```
Comment (seb-odoo) [inline addons/mail/static/tests/new/chat/chat_window_tests.js]: just the syntax change is good, no need to remove the next render

File: addons/mail/static/tests/new/message/message_tests.js [hunk 3]
```diff
@@ -1161,9 +1161,8 @@ QUnit.test(
             model: "res.partner",
             res_id: threadId,
         });
-        const { advanceTime, openFormView } = await start({ hasTimeControl: true });
+        const { openFormView } = await start({ hasTimeControl: true });
         await openFormView("res.partner", threadId);
-        await afterNextRender(() => advanceTime(50 * 1000)); // next fetch of im_status
```
Comment (seb-odoo) [inline addons/mail/static/tests/new/message/message_tests.js:1166]: need to investigate why it doesn't work

**Comment by seb-odoo** [comment]
I think the PR is no longer necessary, you could rebase and see what's left to resolve (if anything).

**Review by alexkuhn — CHANGES_REQUESTED** [review]
This PR looks irrelevant: all tests pass.
Do you agree?

**Pull request closed without merging**